bot/client.js

```diff
@@ -57,4 +57,9 @@ client.on('message', (message) => {
   }
 });
 
+client.on('guildMemberAdd', (member) => {
+  // this will send a message to the user when they enter the discord sevrer
+  member.sendMessage(`Hey ${member.user.username} welcome to the server.`);
```

Comment:
Consider that your implementation solves the problem, and elegantly, but falls just short in that your solution ​within the client.js file does not implement the controller you created. So in effect, any further maintenance will require duplicate efforts in keeping both messages identical.